DFReader.py

```diff
@@ -175,10 +186,8 @@ def find_time_base(self, gps, first_us_stamp):
 
     def type_has_good_TimeMS(self, type):
```

Comment:
I prefer having multiple if statements here.  Once you understand the pattern you can glance down the list of values to see what cases are covered, which is nice.  Assuming this isn't a flake8 change I'd prefer not to have this patch,